src/constants.ts

```diff
@@ -60,3 +60,5 @@ export const TAXONOMY_CUSTOM_FIELD_UNGROUPED_RESOURCE = 'ungrouped';
 export const OLD_SUBJECT_PAGE_REDIRECT_CUSTOM_FIELD = 'old-subject-id';
 
 export const LocaleValues = ['nb', 'nn', 'en'] as const;
+
+export const MastheadHeightPx = 84; // See `misc` in @ndla/core for origin
```

Comment:
Kunne du ikkje brukt mastheadHeight fra core direkte?

Reply:
I core så er det bare en streng som er `84px` så tenkte det var like greit å legge den her.
Kan selvfølgelig oppdatere core til å bruke et tall for å lage strengen though

Reply:
Ikkje gidd.